--- a/src/main/java/ca/spottedleaf/moonrise/common/list/ShortList.java
+++ b/src/main/java/ca/spottedleaf/moonrise/common/list/ShortList.java
@@ -24,14 +_,10 @@
     }
 
     public void setMinCapacity(final int len) {
-        if (len <= 0) {
-            return;
-        }
         final short[] byIndex = this.byIndex;
         if (byIndex.length < len) {
             this.byIndex = Arrays.copyOf(byIndex, len);
         }
-        this.map.ensureCapacity(len);
     }
 
     public boolean add(final short value) {
