--- a/src/main/java/ca/spottedleaf/moonrise/common/time/TickTime.java
+++ b/src/main/java/ca/spottedleaf/moonrise/common/time/TickTime.java
@@ -1,6 +_,6 @@
 package ca.spottedleaf.moonrise.common.time;
 
-import ca.spottedleaf.concurrentutil.util.TimeUtil;
+import ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool;
 
 // All time units are in nanoseconds.
 public final record TickTime(
@@ -57,7 +_,7 @@
      * Returns whether there was a tick that occurred before this one.
      */
     public boolean hasLastTick() {
-        return this.previousTickStart != TimeUtil.DEADLINE_NOT_SET;
+        return this.previousTickStart != SchedulerThreadPool.DEADLINE_NOT_SET;
     }
 
     /*
